source/_integrations/sensor.rest.markdown

```diff
@@ -115,6 +115,11 @@ resource_template:
   description: The resource or endpoint that contains the value with template support.
   required: true
   type: template
+scan_interval:
+  description: Defines number of seconds for polling interval.
+  required: false
+  type: integer
+  default: 15
```

Comment:
This is not an integration option, but a generic platform option.

And it is documented here: <https://www.home-assistant.io/docs/configuration/platform_options/#scan-interval>

It is not specific to this integration, nor is this option provided by this integration.

Closing this PR for that reason.